modules/signing_scriptworker/templates/dep-passwords.json.erb

```diff
@@ -18,6 +18,7 @@
         ["mac-v2-signing11.srv.releng.mdc2.mozilla.com:9110", "<%= scope.function_secret(["signing_server_username"]) %>", "<%= scope.function_secret(["signing_server_dep_password"]) %>", ["macapp"], "signing_server"],
         ["mac-v2-signing12.srv.releng.mdc2.mozilla.com:9110", "<%= scope.function_secret(["signing_server_username"]) %>", "<%= scope.function_secret(["signing_server_dep_password"]) %>", ["macapp"], "signing_server"],
         ["mac-v2-signing13.srv.releng.mdc2.mozilla.com:9110", "<%= scope.function_secret(["signing_server_username"]) %>", "<%= scope.function_secret(["signing_server_dep_password"]) %>", ["macapp"], "signing_server"],
+        ["https://autograph-2.stage.mozaws.net", "<%= scope.function_secret(["autograph_fennec_dep_username"]) %>", "<%= scope.function_secret(["autograph_fennec_dep_password"]) %>", ["autograph_fennec"], "autograph"]
```

Comment:
@JohanLorenzo Am a bit confused as to why I had to use the staging instance. Am looking below and I see the `mar` `dep` counterpart is talking to the production instance. Should we leave this be as is?

Reply:
That's a great question. I'm not too sure what's the scope of "stage" in `autograph-2.stage`. Is it meant to host non-production keys? Or is it meant to test autograph changes? Or both, @g-k?